py-polodb/Cargo.toml

```diff
@@ -19,4 +19,5 @@ crate-type = ["cdylib", "rlib"]
 [dependencies]
 # polodb_core = "5.1.0"
 polodb_core = { path = "../src/polodb_core", default-features = false  }
-pyo3 = { version = "0.22.2", features = ["extension-module"] }
+pyo3 = { version = "0.22.2", features = ["extension-module", "auto-initialize"] }
+bson = "2.0"
```

Comment:
Do we really need this? `polodb_core` has an export bson module: `bson_core::bson`

Reply:
removed